packs/feat-effects/effect-undead-blood-magic.json

```
@@ -0,0 +1,41 @@
+{
+    "_id": "2wCAEtshgSzRuYNK",
+    "img": "systems/pf2e/icons/features/classes/undead.webp",
+    "name": "Effect: Undead Blood Magic",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.classfeatures.7WBZ2kkhZ7JorWu2]{Bloodline: Undead}</p>\n<p>Necromantic energy flows through you or one target. Either you gain temporary Hit Points equal to the spell's rank for 1 round.</p>"
```

```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.classfeatures.7WBZ2kkhZ7JorWu2]{Bloodline: Undead}</p>\n<p>You gain temporary Hit Points equal to the spell's rank for 1 round.</p>"
```
No need for fluff on effects.